plugin/storage/cassandra/spanstore/reader_test.go

```diff
@@ -73,10 +74,19 @@ func TestSpanReaderGetServices(t *testing.T) {
 
 func TestSpanReaderGetOperations(t *testing.T) {
 	withSpanReader(func(r *spanReaderTest) {
-		r.reader.operationNamesReader = func(string) ([]string, error) { return []string{"operation-a"}, nil }
-		s, err := r.reader.GetOperations(context.Background(), "service-x")
+		r.reader.operationNamesReader =
+			func(parameters spanstore.OperationQueryParameters) ([]spanstore.Operation, error) {
+				return []spanstore.Operation{
```

Comment:
nit: you could pull this slice into `expectedOperations` var and reuse it in the assert.